package.json

```diff
@@ -8,7 +8,8 @@
     "dev:next": "npm run lint && next",
     "start": "NODE_ENV=production node server.js",
     "start:next": "next start",
-    "lint": "eslint --fix --ext .gql --ext .graphql --ext .js pages components libraries server helper_scripts",
+    "lint": "eslint --fix --ext .gql --ext .graphql --ext .js pages components containers layouts libraries server helper_scripts",
+    "lint:watch": "esw -w --fix --ext .gql --ext .graphql --ext .js pages components containers layouts libraries server helper_scripts",
```

Comment:
I will search other better options for this. Because there is hot reload on next.js side.